SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -444,6 +449,15 @@ TArray<UClass*> GetAllSupportedClasses()
 			continue;
 		}
 
+		// Avoid processing classes contained in Directories to Never Cook
+		bool bNeverCook = false;
+		if (FDirectoryPath* ContainingDirectory = DirectoriesToNeverCook.FindByPredicate([SupportedClass](FDirectoryPath Directory) {
+			return SupportedClass->GetPathName().StartsWith(Directory.Path);
```

Comment:
Nit: Put `SupportedClass->GetPathName()` into a variable before doing the find.

Reply:
Done